client/components/domains/search-filters/style.scss

```diff
@@ -15,6 +14,9 @@
 
 	.button {
 		align-items: center;
+		border-radius: 3px;
```

Comment:
Could we use the shorthand here? 0 3px 3px 0

Reply:
@shaunandrews You bet! Will make that change now <3